apps-rendering/src/components/Layout/GalleryLayout.tsx

```diff
@@ -23,31 +29,59 @@ const wrapperStyles = css`
 	background-color: ${neutral[7]};
 `;
 
+export const onwardStyles = css`
+	background: ${neutral[97]};
+`;
+
 type Props = {
 	item: Item;
 };
 
-const GalleryLayout: FC<Props> = ({ item, children }) => (
-	<main>
-		<article css={wrapperStyles}>
-			<header css={headerStyles}>
-				<MainMedia
-					mainMedia={item.mainMedia}
-					format={getFormat(item)}
-				/>
-				<Series item={item} />
-				<Headline item={item} />
-				<Standfirst item={item} />
-				<Metadata item={item} />
-				<GalleryCaption
-					mainMedia={item.mainMedia}
-					format={getFormat(item)}
-				/>
-			</header>
-			{children}
-		</article>
-	</main>
-);
+const GalleryLayout: FC<Props> = ({ item, children }) => {
+	const commentContainer = item.commentable
+		? pipe(
+				item.internalShortId,
+				map((id) => (
+					<section
+						css={onwardStyles}
+						id="comments"
+						data-closed={false}
+						data-pillar={themeToPillarString(item.theme)}
+						data-short-id={id}
+					></section>
+				)),
+				withDefault(<></>),
+		  )
+		: null;
+
+	return (
+		<main>
+			<article css={wrapperStyles}>
+				<header css={headerStyles}>
+					<MainMedia
+						mainMedia={item.mainMedia}
+						format={getFormat(item)}
+					/>
+					<Series item={item} />
+					<Headline item={item} />
+					<Standfirst item={item} />
+					<Metadata item={item} />
+					<GalleryCaption
+						mainMedia={item.mainMedia}
+						format={getFormat(item)}
+					/>
+				</header>
+				{children}
+				<Tags item={item} />
+			</article>
+			<section css={onwardStyles}>
+				<RelatedContent item={item} />
+			</section>
+			{commentContainer}
+			<Footer isCcpa={false} format={item} />
```

Comment:
Point by point:
- Yes, that's no problem
- It's in a section tag because the aside is a grid container with an auto margin so to apply the light background colour across the page, I wrapped that grid container in the section. These styles do not need to live inside the layout component but that's why they're there. Thoughts?

Reply:
On the second point, I may be missing something, but I've tried deleting the section tag and it didn't seem to make a difference to the layout? I think the `aside` is full width and has that same light background itself?

Reply:
You're not missing something, I think what I said was true at some point but you're absolutely right